docs/sample_config.yaml

```diff
@@ -314,7 +314,7 @@ listeners:
 #
 # Defaults to `7d`. Set to `null` to disable.
```

Comment:
incidentally, we should probably decide if we use \` or '... we seem to have a mix.